dockerfiles/launcher/launcher_funcs.sh

```diff
@@ -166,6 +166,22 @@ docker_run_with_debug() {
   fi
 }
 
+docker_run_with_volumes_lib() {
+  if has_lib_volume; then
+    docker_run_with_debug -v "${CHE_LIB_VOLUME}":/data/lib "$@"
```

Comment:
The new CLI code is linked to in my previous comment - those links are all pointing to the internals of the new CLI.